movies/src/resources/doc.jsonld

```diff
@@ -6,10 +6,48 @@
   "hydra:description": "This API shows how a basic CRUD (Create, Read, Update, Delete) API can be designed using Hydra Core Vocabulary.",
   "hydra:entrypoint": "/",
   "hydra:supportedClass": [
+    {
+      "@id": "schema:EntryPoint",
+      "@type": "hydra:Class",
+      "hydra:title": "API entrypoint",
+      "hydra:supportedProperty": [
+        {
+          "hydra:property": "hydra:collection",
+          "hydra:title": "Collections",
```

Comment:
In older APIs I have found usage of `hydra:title` and `hydra:description` mixed up with `rdfs:label` and `rdfs:comment`. I am unsure when to use which, or if it is necessary to use both? I stick to hydra vocab for now since this seams to be enough to show up in https://generic.hypermedia.app/

Reply:
Using `hydra:title` and `hydra:description` is may be used as an override for vocabulary specific labels and description, especially helpful for `hydra:supportedProperty`. Generally speaking, my understanding is: when encountered, use hydra specific, otheriwise use `rdfs:label` and/or `rdfs:comment` if available